core/src/main/java/com/linecorp/armeria/client/HttpClientDelegate.java

```diff
@@ -188,6 +193,14 @@ private static boolean sanitizePath(HttpRequest req) {
         return true;
     }
 
+    private static void handleEarlyException(ClientRequestContext ctx,
+                                             HttpRequest req, Throwable cause) {
+        req.abort();
+        final RequestLogBuilder logBuilder = ctx.logBuilder();
+        logBuilder.endRequest(cause);
```

Comment:
I'm curious if `req.abort()` will trigger `logBuilder.endRequest()` anyway. Could you check?

Reply:
Unfortunately, `req.abort()` does not trigger it unless it is subscribed by the `HttpRequestSubscriber`. So this is needed it, or I can change the `ReqeustLogBuilder.endResponse(cause)` to end request when the `Request` has not started. What do you think?

Reply:
Thanks for explanation. Let's leave it as it is.